paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -56,4 +58,15 @@ open class PrepareResourcesTask : DefaultTask() {
         }
     throw IllegalStateException("No source sets available")
   }
+
+  private fun Project.targetSdkVersion(): String {
+    val androidExtension = extensions.getByType(BaseExtension::class.java)
+    return androidExtension.compileSdkVersion.substringAfter(
+        "android-", DEFAULT_COMPILE_SDK_VERSION.toString()
```

Comment:
This is not guaranteed to be the format and it's not guaranteed to contain a number. It can be any arbitrary string which is then looked up in the `platforms/` dir as a folder name.

Reply:
Originally, I was attempting to use manifest.targetSdk but when that returned null, i started using BaseExtension.getCompileSdkVersion whose underlying target member is set via one of the following methods:

```
    /** @see #getCompileSdkVersion() */
    public void compileSdkVersion(String version) {
        checkWritability();
        this.target = version;
    }

    /** @see #getCompileSdkVersion() */
    public void compileSdkVersion(int apiLevel) {
        compileSdkVersion("android-" + apiLevel);
    }

    public void setCompileSdkVersion(int apiLevel) {
        compileSdkVersion(apiLevel);
    }

    public void setCompileSdkVersion(String target) {
        compileSdkVersion(target);
    }
```